src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -55,6 +55,14 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     setActiveOption(undefined);
   }, []);
 
+  const onPollSubmit = useCallback(
+    (data) => {
+      createPoll(data);
+      unselectOption();
+    },
+    [createPoll, unselectOption]
+  );
```

Comment:
[OOS] Calling this OOS as it's a bit more involved than the specific fix intended here, but `createPoll` should really be returning it's `Promise`, and then we would be able to use `createPoll.then(() => unselectOption())` here or similar. Then it might make sense to use `useAsync` here with `onPollSubmit` so that we can easily capture/surface the loading/error/etc states from it.

Reply:
Actually, it would be nice to add `@debt` comments as part of this PR calling this out though, as a future reminder/etc.

Reply:
fixed in 7126439c4407614d55434c09a72581140e6c94b8